src/cascadia/TerminalApp/App.cpp

```diff
@@ -405,6 +405,13 @@ namespace winrt::TerminalApp::implementation
         _newTabButton.Flyout(newTabFlyout);
     }
 
+    void App::_OpenDropdown()
+    {
+        Controls::Primitives::FlyoutShowOptions options{};

```

Comment:
What happens when the dropdown is already visible?

Reply:
The keybinding is ignored and the <kbd>Space</kbd> key will open the first profile in the list

Reply:
Oh that's an actually good behavior.